src/test/java/io/github/glytching/junit/extension/random/RandomBeansExtensionFieldTest.java

```diff
@@ -148,4 +159,11 @@ public void willInjectANewRandomValueEachTime() {
       anyStrings.add(anyString);
     }
   }
+
+  @RepeatedTest(5)
+  public void willNotInjectANewRandomValueEachTime() {
```

Comment:
Could you rename this to make it clear that it relates to static fields? e.g. `willNotInjectANewRandomValueEachTimeForAStaticField`.

Reply:
Thanks! Done, I applied your suggestion!